coderdojochi/views.py

```diff
@@ -400,8 +400,9 @@ def session_sign_up(request, year, month, day, slug, session_id, student_id=Fals
                 session_obj.mentors.add(mentor)
         else:
             if user_signed_up:
-                order = get_object_or_404(Order, student=student, session=session_obj)
-                order.delete()
+                order = get_object_or_404(Order, student=student, session=session_obj, active=True)
```

Comment:
@bkellgren That's what I thought but it wasn't working when I tried it. Let me retest.